src/rpc/server.ts

```diff
@@ -621,6 +620,7 @@ export class RpcServer {
    * transaction completion and return a definitive state of success or failure.
    *
    * @param {string} hash   the transaction you're polling for
+   * @param {RpcServer.PollingOptions} [opts] (optional) polling options
```

Comment:
```suggestion
   * @param {[RpcServer.PollingOptions]} [opts] polling options
```

I think this makes them marked accordingly in jsdoc gen